go/vt/vttablet/onlineddl/executor.go

```diff
@@ -1561,6 +1581,75 @@ func (e *Executor) scheduleNextMigration(ctx context.Context) error {
 	return err
 }
 
+// reviewQueuedMigrations iterates queued migrations and sees if any information needs to be updated
+func (e *Executor) reviewQueuedMigrations(ctx context.Context) error {
+	e.migrationMutex.Lock()
+	defer e.migrationMutex.Unlock()
+
+	{
```

Comment:
Can we remove this new block? I am maybe missing its added value

Reply:
Removed! I was thinking there could be multiple "sections" for reviewing the migrations, and so each would be in a separate block, for clarity. But right now there was just this one...

Reply:
Okay, I understand now